services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -390,28 +410,40 @@ async def list_projects(  # pylint: disable=too-many-arguments
                         if not include_hidden
                         else sa.text("")
                     )
-                    & (
-                        (projects.c.prj_owner == user_id)
-                        | sa.text(
-                            f"jsonb_exists_any(access_rights_subquery.access_rights, {assemble_array_groups(user_groups)})"
-                        )
-                    )
                     & (
                         (projects_to_products.c.product_name == product_name)
                         # This was added for backward compatibility, including old projects not in the projects_to_products table.
                         | (projects_to_products.c.product_name.is_(None))
                     )
+                    & (
+                        projects_to_folders.c.folder_id == folder_id
+                        if folder_id
```

Comment:
n When explictly checking for None please use `is None`. Let's avoid the possibility of ending up with values like `0, "", {}, []`, even though this is not the case.

Reply:
Hmm `folder_id` can be just positive integer or None (which is already guaranteed at this point). But you are right generally this is a painful thing so in different scenario we need to be more careful.